translations/he/LC_MESSAGES/messages.po

```diff
@@ -194,6 +194,35 @@ msgstr "תאונות"
 msgid "Fatal, severe and light accidents count in the specified location."
 msgstr "כמות התאונות הקטלניות, הקשות והקלות במקטע בפרק הזמן הנבחר."
 
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:48
+msgid "Most severe accidents in segment"
+msgstr "תאונות חמורות במקטע"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:54
+msgid "Severe accidents in street"
+msgstr "תאונות חמורות ברחוב"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:56
+msgid "in "
+msgstr "ב"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:154
+msgid "accident_severity"
+msgstr "חומרת תאונה"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:155
+msgid "type"
+msgstr "סוג"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:169
+#, fuzzy
```

Comment:
The fuzzy flag prevent the use of this specific translation - we might need to recompile after removing the fuzzy flag, is this intentional?

https://codehunter.cc/a/flask/flask-babel-translations-de-lc-messages-messages-po-is-marked-as-fuzzy-skipping